core/tests/protractor_utils/editor.js

```diff
@@ -974,13 +974,32 @@ var _runFromHistoryTab = function(callbackFunction) {
 var _selectComparedVersions = function(v1, v2) {
   var v1Position = null;
   var v2Position = null;
-  element.all(by.css('.protractor-test-history-v1-selector')).first()
-      .getAttribute('value').then(function(versionNumber) {
+  element.all(by.css('.protractor-test-history-checkbox-selector')).count()
+      .then(function(versionNumber) {
+    if (v1 < 0) {
+      throw Error('In editor._selectComparedVersions(' + v1 + '),' +
```

Comment:
shouldn't this be 
    throw Error('In editor._selectComparedVersions(' + v1 + ', ' + v2 + '),' +

ditto for the ones below.